### PR TITLE
Attempt to fix bug #80

### PR DESCRIPTION
**Pull request by victorct-pulsarup**
Attempt to fix bug #80

### CODE REVIEW

File: django_celery_beat/schedulers.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def _default_now(self):
         return now
 
     def __next__(self):
-        self.model.last_run_at = self.app.now()
+        self.model.last_run_at = self._default_now()
```
Comment (liquidpele) [inline django_celery_beat/schedulers.py]: What was self.app.now() returning?  UTC, system time without timezone, system time with timezone?

**Comment by liquidpele** [comment]
@victorct-pulsarup Hi there, can you merge in latest master branch to your branch to update the PR?  We've recently fixed the tests so it'll set your PR as passing.

File: django_celery_beat/schedulers.py (hunk 1 above)
Comment (victorct-pulsarup) [reply django_celery_beat/schedulers.py]: Timezone awareness datetime

**Comment by liquidpele** [comment]
Tests pass, looks good.  thanks for the contribution!

**Comment by victorct-pulsarup** [comment]
Tests passed, sorry for the delay, been busy at work.

Awesome library by the way!

**Pull request merged**